wiki_docs/Create How_GTM_works

```diff
@@ -0,0 +1,47 @@
+User can add or manage the AdWords, Google analytics, Flood-light and non-Google tags using the GTM.
+
+How it works
+  Followings are main components of the GTM:
+
+Account
+Each account can contains multiple containers
+  "One account per company/organization"
+
+Container
+A container includes the tracking tags (e.g. Google Analytics, AdWords Conversion Tracking tags) that you want to use on the website.
+  "One container for one website"
+
+Tags
+Tags are snippets of code on your site that collect data
+  "A tag must have at least one rule in order to fire"
+Google Tag Manager supports following tag types which can be use directly:
+  1. AdWords Conversion Tracking
+  2. DoubleClick Floodlight Counter
+  3. DoubleClick Floodlight Sales
+  4. Google Analytics
+  5. GDN Remarketing
+
+There are other two custom tags available which allows you to implement custom tags which are not available in above list:
+  1. Custom Image Tag
+  2. Custom HTML Tag
+  
+Rules
+The rule(s) attached to a tag govern when the tag is fired or not fired.
+  "A rule that blocks a tag from firing always overrides a firing rule"
+Syntax: [Macro] [Operator] [Value]
+Example: url equals example.com/index.html
+
+Macros
+Macros supply dynamic data to tags and conditions.
+  "Macros are name-value pairs for which the value is populated during runtime"
+
+  Types of Macro
+  
+  1. Constant String
+  2. JavaScript Variable
+  3. Data Layer
+  4. DOM Text
+  5. Dom attribute
+  6. URL
+  7. Custom Event
+  8. HTTP Referrer
```

Comment:
Hi @grprtkal ,
I have added a section "how to add a tag". Please have a look into this section.
Please review this section and elaborate things if you want to add more.

Cheers,
Amit Kumar Jaiswal